model/shared/src/main/scala/hmda/model/institution/InstitutionType.scala

```diff
@@ -7,13 +7,13 @@ import enumeratum.{ Enum, EnumEntry }
  */
 sealed trait DepositoryType extends EnumEntry
 
-// FIXME: This could also be modeled as `isDepository: Option[Boolean]`.  Which is cleaner?
 object DepositoryType extends Enum[DepositoryType] {
 
   val values = findValues
 
   case object Depository extends DepositoryType
   case object NonDepository extends DepositoryType
+  case object UndeterminedDepositoryType extends DepositoryType
```

Comment:
Do we want institution records in our system where we can't determine the `DepositoryType` and `InstitutionType`.  If you don't have them these set, you cannot derive the `respondentId` or do the related edits. We previously decided that `DepositoryType` would always present, and that's why we changed it to non-Optional (https://github.com/cfpb/hmda-platform/commit/abd7314d7e107b08fd2baa28bf461c74c1580e6d)

I think the logic in [Q033](https://github.com/cfpb/hmda-platform/blob/8341ef8b181f321fe9956ef842e92dc7158667b1/validation/src/main/scala/hmda/validation/rules/ts/quality/Q033.scala#L17) and [V110](https://github.com/cfpb/hmda-platform/blob/8341ef8b181f321fe9956ef842e92dc7158667b1/validation/src/main/scala/hmda/validation/rules/ts/validity/V110.scala#L17) assumes only known types will be set.  What do we do in these edits if `UndeterminedInstitutionType` shows up?

I guess I'm just generally concerned with all of these _undetermined_ types.  It feels like we're letting bad data in that will lead to downstream consequences.  Can we just not put these institutions in our system until we've resolved whatever upstream data sources that are causing these values to be _undetermined_?